### ISSUE TITLE
How to deal with the history and tagged data

### ISSUE THREAD

**Issue by emilienschultz** [emilienschultz/activetigger#121]
Situation identified by Leo : 
- to select tagged element, I have to keep the history of previous elements (to only select them once to annotate, otherwise it is always the same)
- but if i tag all the untagged element, it means they are all in the history
- so they can't be selected again with the tagged sample

The solution is to clean the history. But I don't see how to do it automatically. An idea @eollion @jboelaert ?

**Comment by jboelaert** [comment]
I'm not sure I understand the question exactly, but if I do it is a tricky one indeed. What I did in R Tigger was to keep a running memory, per session, of all the elements that have just been tagged (from untagged) or re-tagged (from tagged), and to prevent these elements from being presented again to the user (during this session).
Does this answer the question? Is it doable in pyTigger?

**Comment by emilienschultz** [comment]
Yes. I do that. But the case is : if you tagged it in one session and in the same you want to review the tagged element, you can't access them, except if you click on "reset history". So it works as you said (so I guess there is the same problem in ractivetigger), but with this potential problem for unexperimented users (who doesn't know about history).

**Comment by emilienschultz** [comment]
Make it visible in the navigation with a tooltip
